Apps/W1/Shopify/app/src/Logs/Codeunits/ShpfySkipRecordMgt.Codeunit.al

```diff
@@ -0,0 +1,44 @@
+namespace Microsoft.Integration.Shopify;
+
+/// <summary>
+/// Codeunit Shpfy Skip Record Mgt. (ID 30313).
+/// </summary>
+codeunit 30313 "Shpfy Skip Record Mgt."
```

Comment:
Let's rename this to "Shpfy Skipped Record"

Mgt. does not seem very necessary

Reply:
done